### PR TITLE
Add load on demand

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

File: controls/collectionview/load-on-demand/overview.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+page_title: .NET MAUI CollectionView Documentation - Load On Demand
+description: Learn how to load data on demand in the Telerik UI for .NET MAUI CollectionView automatically or manually.
+position: 0
+slug: collectionview-load-on-demand
+tags: loading data, .net maui, maui, collectionview, load data on demand
+---
+
+# .NET MAUI CollectionView Load Data on Demand
+
+The Load Data on Demand feature optimizes the performance of the Telerik UI for .NET MAUI CollectionView control when operating with a large amount of items. The load on demand functionality allows you to load incremental data&mdash;either by a Button (manual) or by scrolling towards end (automatic).
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:12]:
```suggestion
The Load Data on Demand feature optimizes the performance of the Telerik UI for .NET MAUI CollectionView control when operating with a large amount of items. The load-on-demand functionality allows you to load incremental data&mdash;either by a Button (manual) or by scrolling towards end (automatic).
```

File: controls/collectionview/load-on-demand/overview.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Configuration
+
+Use the following configuration options to:
+
+1. Control the `LoadOnDemand` feature by using the following properties:
+
+* `IsLoadOnDemandEnabled` (`bool`)&mdash;Specifies whether the load on demand functionality is enabled. The default value is `false`.
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:20]:
```suggestion
* `IsLoadOnDemandEnabled` (`bool`)&mdash;Specifies whether the load-on-demand functionality is enabled. The default value is `false`.
```

File: controls/collectionview/load-on-demand/overview.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+>tip The `LoadOnDemandBufferItemsCount` works with `LoadOnDemandMode.Automatic`.
+
+3. Customize the appearance of the automatic and manual UI element.
+
+You can customize the appearance on the automatic and manual elements by using the following templates:
+
+* `AutomaticLoadOnDemandTemplate` (`DataTemplate`)&mdash;Specifies the template of the view visualized for `Automatic` loading.
+* `ManualLoadOnDemandTemplate` (`DataTemplate`)&mdash;Specifies the template of the view visualized for `Manual` loading.
+
+For sample examples and templates explanation review the [LoadOnDemand Templates]({%slug collectionview-load-on-demand-templates%}) article.
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:39]:
```suggestion
For more information about the templates, including examples, review the [LoadOnDemand Templates]({%slug collectionview-load-on-demand-templates%}) article.
```

File: controls/collectionview/load-on-demand/overview.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Methods to Load Data on Demand
+
+Use the following options to load data on demand:
+
+* Using the [`LoadOnDemandCollection`]({%slug collectionview-load-on-demand-collection%}).
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:45]:
```suggestion
* [`LoadOnDemandCollection`]({%slug collectionview-load-on-demand-collection%})
```

File: controls/collectionview/load-on-demand/overview.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+* Using the [`LoadOnDemandCommand`]({%slug collectionview-load-on-demand-command%}).
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:46]:
```suggestion
* [`LoadOnDemandCommand`]({%slug collectionview-load-on-demand-command%})
```

File: controls/collectionview/load-on-demand/overview.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+* Using the [`LoadOnDemand` event]({%slug collectionview-load-on-demand-event%}).
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/overview.md:47]:
```suggestion
* [`LoadOnDemand` event]({%slug collectionview-load-on-demand-event%})
```

File: controls/collectionview/load-on-demand/collection.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+```XAML
+xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
+```
+
+**4.** Create a `ViewModel` and use the `LoadOnDemandCollection` as a type of the property bound to the `RadCollectionView.ItemsSource`:
+
+<snippet id='collectionview-loadondemand-collection-viewmodel' />
+
+This is the result:
+
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/collection.md:39]: Where is the result? :)

File: controls/collectionview/load-on-demand/templates.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+page_title: .NET MAUI CollectionView Documentation - Load On Demand Templates
+description: Learn how to customize the default automatic and manual loading indicator in the Telerik UI for .NET MAUI CollectionView when using the load on demand feature.
+position: 4
+slug: collectionview-load-on-demand-templates
+tags: loading data, .net maui, maui, collectionview, load data on demand, loading templates
+---
+
+# .NET MAUI CollectionView LoadOnDemand Automatic and Manual Templates
+
+You can customize the appearance on the automatic and manual elements by using the following templates:
```
Comment (ikoevska) [inline controls/collectionview/load-on-demand/templates.md:12]:
```suggestion
You can customize the appearance of the automatic and manual elements by using the following templates:
```

**Review by ikoevska — CHANGES_REQUESTED** [review]
Please, fix the navigation title in the config file, as per this error:

> No title for `controls/collectionview/load-on-demand` path. Consider fixing in _config.yml - navigation.

Also, all "this is the result" seem to be empty. Are these happening in a separate PR?

File: controls/collectionview/load-on-demand/collection.md (hunk 7 above)
Comment (didiyordanova) [reply controls/collectionview/load-on-demand/collection.md:39]: with the upcoming commit ;)

**Comment by didiyordanova** [comment]
@ikoevska the config.yml will be updated when all features for the CollectionView are merged,. Then we will review the features order and address this feedback. For the release this will be addressed.

**Pull request merged**